src/sass/_Fabric.Icons.scss

```diff
@@ -347,3 +347,20 @@
     z-index: $ms-zIndex-back;
   }
 }
+
+// Fabric Core Icon variables
+$ms-Icon-12: 12px;
+$ms-Icon-16: 16px;
+$ms-Icon-20: 20px;
```

Comment:
I believe the pattern is to lowercase variable names.
$ms-icon-12
$ms-icon-16
$ms-icon-20